APIKit/APIKit.swift

```diff
@@ -72,47 +75,9 @@ public class API: NSObject, NSURLSessionDelegate, NSURLSessionDataDelegate {
     public class func responseBodyParser() -> ResponseBodyParser {
         return .JSON(readingOptions: nil)
     }
-    
-    public class func URLSessionConfiguration() -> NSURLSessionConfiguration {
-        return NSURLSessionConfiguration.defaultSessionConfiguration()
-    }
-    
-    public class func URLSessionDelegateQueue() -> NSOperationQueue? {
-        // nil indicates NSURLSession creates its own serial operation queue.
-        // see doc of NSURLSession.init(configuration:delegate:delegateQueue:) for more details.
-        return nil
-    }
-    
-    // prevent instantiation
-    override private init() {
```

Comment:
Ah, I got what you meant: pure Swift class which defines no initializer cannot be initialized anyway :+1: